src/scoring.go

```diff
@@ -64,79 +84,96 @@ func checkConfigData() {
+		if debugEnabled {
+			infoPrint(fmt.Sprint("Result of last pass check was ", status))
 		}
+		passStatus = append(passStatus, passItemStatus)
+	}
 
-		// For multiple pass conditions, will only be true if ALL of them are
-		for _, result := range passStatus {
-			status = status && result
-			if !status {
-				break
-			}
+	// For multiple pass conditions, will only be true if ALL of them are
+	for _, result := range passStatus {
+		status = status && result
+		if !status {
+			break
 		}
+	}
+	if debugEnabled {
+		infoPrint(fmt.Sprint("Result of all pass check was ", status))
+	}
+
+	// If a PassOverride succeeds, that overrides the Pass checks
+	for _, condition := range check.PassOverride {
+		passOverrideStatus := processCheckWrapper(&check, condition.Type, condition.Arg1, condition.Arg2, condition.Arg3)
 		if debugEnabled {
-			infoPrint(fmt.Sprint("Result of all pass check was ", status))
+			infoPrint(fmt.Sprint("Result of pass override was ", passOverrideStatus))
 		}
-
-		// If a PassOverride succeeds, that overrides the Pass checks
-		for _, condition := range check.PassOverride {
-			passOverrideStatus := processCheckWrapper(&check, condition.Type, condition.Arg1, condition.Arg2, condition.Arg3)
-			if debugEnabled {
-				infoPrint(fmt.Sprint("Result of pass override was ", passOverrideStatus))
-			}
-			if passOverrideStatus {
-				status = true
-				break
-			}
+		if passOverrideStatus {
+			status = true
+			break
 		}
-		for _, condition := range check.Fail {
-			failStatus := processCheckWrapper(&check, condition.Type, condition.Arg1, condition.Arg2, condition.Arg3)
-			if debugEnabled {
-				infoPrint(fmt.Sprint("Result of fail check was ", failStatus))
-			}
-			if failStatus {
-				status = false
-				break
-			}
+	}
+	for _, condition := range check.Fail {
+		failStatus := processCheckWrapper(&check, condition.Type, condition.Arg1, condition.Arg2, condition.Arg3)
+		if debugEnabled {
+			infoPrint(fmt.Sprint("Result of fail check was ", failStatus))
 		}
-		if check.Points >= 0 {
-			if status {
-				if verboseEnabled {
-					passPrint(fmt.Sprintf("Check passed: %s - %d pts", check.Message, check.Points))
-				}
-				mc.Image.Points = append(mc.Image.Points, scoreItem{check.Message, check.Points})
-				mc.Image.Score += check.Points
-				mc.Image.Contribs += check.Points
+		if failStatus {
+			status = false
+			break
+		}
+	}
+	if check.Points >= 0 {
+		if status {
+			if verboseEnabled {
+				passPrint(fmt.Sprintf("Check passed: %s - %d pts", check.Message, check.Points))
 			}
-		} else {
-			if status {
-				if verboseEnabled {
-					failPrint(fmt.Sprintf("Penalty triggered: %s - %d pts", check.Message, check.Points))
-				}
-				mc.Image.Penalties = append(mc.Image.Penalties, scoreItem{check.Message, check.Points})
-				mc.Image.Score += check.Points
-				mc.Image.Detracts += check.Points
+			mc.Image.Points = append(mc.Image.Points, scoreItem{check.Message, check.Points})
```

Comment:
extra spaces/tabs

Reply:
I'm not sure what you mean, I `gofmt`'d it before pushing and it looks alright (?) to me.